Dockerfile.surveyor

```diff
@@ -0,0 +1,21 @@
+FROM python:3.5-slim
+
+RUN groupadd user && useradd --create-home --home-dir /home/user -g user user
+WORKDIR /home/user
+
+ENV CELERY_VERSION 4.0.2
+
+COPY foreman/requirements.txt .
+
+RUN pip install -r requirements.txt
+
+COPY data_models/dist/data-refinery-models-* data_models/
```

Comment:
> Would it make sense for them to be packages on PyPI that we can install?

Yes definitely. I actually had made a PyPI package for this and was installing it but that necessitated having to change the version each time I wanted to make a change even while developing. Once the data models are a bit more stable and I'm ready to release an initial version I will change this back to installing from PyPI.